templates/base.html

```diff
@@ -55,6 +55,8 @@
 
   <link href="{{ SITEURL }}/{{ THEME_STATIC_DIR }}/css/style.css" type="text/css" rel="stylesheet" />
 
+    <link href="{{ SITEURL }}/{{ THEME_STATIC_DIR }}/css/mystyles.css" type="text/css" rel="stylesheet" />
```

Comment:
If you want to add your own css, you can use the property called `CSS_OVERRIDE`  refer https://github.com/arulrajnet/attila#user-defined-css